RLTest/loader.py

```diff
@@ -3,6 +3,7 @@
 import sys
 import imp
 import inspect
+from RLTest.utils import Colors
```

Comment:
IMHO We should really remove our on colors and just use the package that does this.

Reply:
It's a bit out of context for this PR, but yes, we should.